MapboxNavigation/NavigationMapView.swift

```diff
@@ -258,9 +258,7 @@ open class NavigationMapView: MLNMapView, UIGestureRecognizerDelegate {
         }
     }
    
-    override open func mapViewDidFinishRenderingFrameFullyRendered(_ fullyRendered: Bool, frameEncodingTime: Double, frameRenderingTime: Double) {
-        super.mapViewDidFinishRenderingFrameFullyRendered(fullyRendered, frameEncodingTime: frameEncodingTime, frameRenderingTime: frameRenderingTime)
-        
+    func updateCourseTrackingAfterDidFinishRenderingFrame() {
```

Comment:
What's the reason for removing the override and then calling this from elsewhere? For me the override function is more logical

Reply:
It's so that we don't accidentally break the puck animation again, like what I fixed in #53.

I tried to explain the reasoning in this issue description — the method we were overriding was a private method. I patched it up in #53, but because it's still a private method it could break again. #53 has some more details which might be helpful for context.

Reply:
Got it, thanks for linking the PR, helps a lot to understand.